internal/pkg/cmd/flags.go

```diff
@@ -183,7 +183,7 @@ func AutocompleteEnvironments(v1Client *ccloudv1.Client, v2Client *ccloudv2.Clie
 		return nil
 	}
 
-	if auditLog := user.GetOrganization().GetAuditLog(); auditLog != nil {
+	if auditLog := user.GetOrganization().GetAuditLog(); auditLog != nil && auditLog.GetAccountId() != "" {
```

Comment:
This looks like a typo?
```suggestion
	if auditLog := user.GetOrganization().GetAuditLog().GetServiceAccountId() != 0 {
```